.github/workflows/run-snapshot-generation.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Gradle - Record snapshots with Paparazzi
         id: testStep
-        run: ./gradlew clean components:recordPaparazziDebug
+        run: ./gradlew components:recordPaparazziDebug
```

Comment:
info: we don't need to clean anything, because we just started the job so no build artifacts are created yet